src/bci_build/package.py

```diff
@@ -2156,7 +2156,13 @@ def _create_php_bci(
                 "ca-certificates-mozilla-prebuilt",
             )
         ],
-        config_sh_script="sed -i 's|/bin/bash|/bin/sh|' /etc/passwd",
+        config_sh_script=textwrap.dedent(
+            """
+            sed -i 's|/bin/bash|/bin/sh|' /etc/passwd
+            # Will be recreated by the next rpm(1) run as root user
+            rm -v /usr/lib/sysimage/rpm/Index.db
```

Comment:
side note: will this really work as expected with rpm not in the container image?

Reply:
> If this is the case for all images, shouldn't we do this for every image then?

we can theoretically do it for all images, but be aware that it breaks non-root usage (so images that are derived  or use a USER and expect to be able to run rpm it would be broken). I would however want to do it in a limited-rollout fashion first for the image where we see the biggest relative benefit.

Reply:
> side note: will this really work as expected with rpm not in the container image?

well, thats why I submitted it for busybox  image only - even if users derive from it they can not install additional packages so the rpm database is not an issue - we just need the Packages.db to not break the tracking.

Reply:
Makes sense, thank you!